### PR TITLE
Fix buggy_function.py

### PR DESCRIPTION
**Pull request by BSGalvan**
Also add .gitignore and improve the README for readability.

### CODE REVIEW

File: participants/Saiguhan/doc/README.md [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
+You can use these functions from the python interpreter, assuming you've installed this
+library. Another example of a common function is as follows:
 
-    >>> fibonnaccci(100)
+    >>> fibonacci(100)
     [0, 1, 1, 2, 3, 5, 8, 13, 21, 34, 55, 89]
 
-If you can't use it, its kind of your problem, not mine!
+If you find a problem with the code, documentation or implementation, please raise an
+issue on the [issue tracker](https://github.com/AstroHackWeek/2022_contributing_to_open_source/issues).
```
Comment (criselsuarez) [inline participants/Saiguhan/doc/README.md:14]: Nice to add the issue link to the README

File: participants/Saiguhan/pr_tutorial/buggy_function.py [hunk 2]
```diff
@@ -20,13 +20,13 @@ def angle_to_sexigesimal(angle_in_degrees, decimals=3):
...
+    hours_num = angle_in_degrees*24/360
     hours = math.floor(hours_num)
 
     min_num = (hours_num - hours)*60
     minutes = math.floor(min_num)
 
     seconds = (min_num - minutes)*60
 
-    format_string = '{}:{}:{:.' + str(decimals) + 'f}'
+    format_string = "{}:{}:{:.' + str(decimals) + 'f}"
```
Comment (criselsuarez) [inline participants/Saiguhan/pr_tutorial/buggy_function.py]: Nice catch of the single quotes!

**Review by eringcox — APPROVED** [review]
These changes look good!

**Review by criselsuarez — APPROVED** [review]
Great job with the README and buggy function!

**Comment by ceb8** [comment]
Thanks! 😄

**Pull request merged**